torch/optim/lr_scheduler.py

```diff
@@ -59,9 +78,26 @@ def load_state_dict(self, state_dict):
         """
         self.__dict__.update(state_dict)
 
-    def get_lr(self):
+    def get_computed_values(self):
+        # Return last computed learning rate by current scheduler
+        return self._last_computed_values
+
+    def _compute_values(self):
+        # NOTE [LR Chainable Form] Compute learning rate using chainable form of the scheduler
+        raise NotImplementedError
+
+    def _compute_values_closed_form(self):
+        # NOTE [LR Closed Form] Compute learning rate using closed form of the scheduler
         raise NotImplementedError
 
+    def get_lr(self):
+        warnings.warn(
+            "get_computed_values is now the supported method to obtain the last "
+            "computed learning rate. get_lr now returns the same, but will be "
+            "deprecated in favor of get_computed_values.", DeprecationWarning
```

Comment:
When I'm told that "instead of using `f`, I should use `g`" where they have exactly the same meaning, I tend to also like to know why a renaming happened at all. Is there actually a subtle semantics change? Is the old name bad and the new name more correctly says what happens? Just saying "foo is the new way of doing bar, bar returns the same thing" sounds like you've just been moving deck chairs around.

Reply:
Good point. I'll add more details, like
```
      "The behavior of schedulers has been changed from their closed form to their "                                                                                                                                                     
      "chainable form. As such, the epoch parameter no longer applies, and is being "                                                                                                                                                    
      "deprecated. During the deprecation, if epoch is different from None, the closed "                                                                                                                                                 
      "form is used whenever available instead of the chainable form." 
```